app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def oauth_login_path
```

Comment:
Ebben az esetben akkor erdemesebb az `ApplicationController`be tenni ezt a metodust es inkabb ott megjelolni, hogy ez egy helper metodus es view-kban is akarod hasznalni. Es akkor nem kell `include ApplicationHelper` minden controllerben ahol kell ez.

http://api.rubyonrails.org/v4.2.7/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method